control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources.go

```diff
@@ -930,6 +931,24 @@ func (r *reconciler) reconcileIngressController(ctx context.Context, hcp *hyperv
 		}
 	}
 
+	if hcp.Spec.Platform.Type == hyperv1.OpenStackPlatform {
+		ingressFloatingIP := hcp.Spec.Platform.OpenStack.IngressFloatingIP
+		ingressProvider := hcp.Spec.Platform.OpenStack.IngressProvider
+		if err := openstackutil.ValidateIngressOptions(ingressProvider, ingressFloatingIP); err != nil {
```

Comment:
why do we need to validate it again if it was already validate at `cmd/cluster/openstack/create.go`?
Is it because the hostedcluster could be created without cmdline?

Reply:
yeah exactly, I thought it'll be nice to avoid controller errors in case someone creates the cluster via manifests.